### DISCUSSION TITLE
Pressing links in source files.

### DISCUSSION THREAD

**Discussion by noahra** [LazyVim/LazyVim#1451] [General]
This is a noobie question, but is there a plugin or an easy way to press, for example, <iostream> declaration inside a file to peer into the contents of that library?

```
#include <iostream> // <-- Is there a plugin for just pressing on iostream to see view its contents?

int main() {
  // code
 // ....
    }
  }
}

```

**Comment by amaanq** [comment]
if you have an lsp setup (clangd w/ the lang extra), gd will work

**Comment by dpetka2001** [comment]
You can also give a look into [glance.nvim](https://github.com/DNLHC/glance.nvim). It tries to emulate the UI of VsCode to peek into definitions/references. But you have to have a working LSP setup for clangd as @amaanq mentioned.

**Reply by egolearner** [reply]
https://github.com/rmagatti/goto-preview is worthy trying once you have a functional LSP setup.
